### DISCUSSION TITLE
Apache NetBeans 23 release candidates

### DISCUSSION THREAD

**Discussion by ebarboni** [apache/netbeans#7615] [General]
Release candidates for the next release of Apache NetBeans will be available weekly until release. Your help in testing, reporting and verifying issues would be highly appreciated by the whole Apache NetBeans community.

Changes from NetBeans 22 can be seen at https://github.com/apache/netbeans/releases/tag/23-rc1

## Binary / source zip

Binary and source zips are available from -

https://nightlies.apache.org/netbeans/candidate/netbeans/

## Installers

Unsigned Windows, Linux DEB and Linux RPM installers are available from -

https://nightlies.apache.org/netbeans/candidate/installers/

NB. We are working on making other installers available for release candidates from the same location.

## VSCode plugin

The VSCode plugin is available from - 

https://nightlies.apache.org/netbeans/candidate/vsix/

## IMPORTANT

These release candidates are pre-release software, and not official releases of Apache NetBeans.

**Comment by mbien** [comment]
rc2 is built, changes since rc1: https://github.com/apache/netbeans/releases/tag/23-rc2

**Comment by mbien** [comment]
rc3 is built, changes since rc2: https://github.com/apache/netbeans/releases/tag/23-rc3

**Comment by mbien** [comment]
closing, NetBeans 23 released -> https://netbeans.apache.org/download/
